tls/s2n_client_hello.c

```diff
@@ -141,6 +147,12 @@ int s2n_client_hello_send(struct s2n_connection *conn)
     /* Write the extensions */
     GUARD(s2n_client_extensions_send(conn, out));
 
+    /* Default our signature digest algorithms */
```

Comment:
Should the client end be setting the defaults here? Aren't we dependent on parsing the {hash,sign} algorithm used by the server in ServerKeyExchange?